integration/tests/epochs/suite.go

```diff
@@ -380,8 +380,15 @@ func (s *Suite) getTestContainerName(role flow.Role) string {
 // and checks that the info.NodeID is in both list
 func (s *Suite) assertNodeApprovedAndProposed(ctx context.Context, env templates.Environment, info *StakedNodeOperationInfo) {
 	// ensure node ID in approved list
-	approvedNodes := s.ExecuteReadApprovedNodesScript(ctx, env)
-	require.Containsf(s.T(), approvedNodes.(cadence.Array).Values, cadence.String(info.NodeID.String()), "expected new node to be in approved nodes list: %x", info.NodeID)
+	//approvedNodes := s.ExecuteReadApprovedNodesScript(ctx, env)
+	//require.Containsf(s.T(), approvedNodes.(cadence.Array).Values, cadence.String(info.NodeID.String()), "expected new node to be in approved nodes list: %x", info.NodeID)
```

Comment:
is this expected to be commented? if the other one is a replacement do we still need it ?

Reply:
yes its expected to be commented from here: https://github.com/onflow/flow-go/pull/4227/commits/159ce81b4bfab8a2be1b41c3929fbd3e79b5e4e3

but it would be better to clean it up in a separate PR. I'll open that against master